projects/aca-folder-rules/src/lib/rules-list/rules-list.ui-component.html

```diff
@@ -0,0 +1,3 @@
+<div class="aca-rules-list">
+  <aca-rule *ngFor="let rule of rules" [rule]="rule" (click)="onRuleClicked(rule)"></aca-rule>
```

Comment:
Is there any visual change (e.g. change of background colour) to indicate a selected rule?